src/scipp/core/operations.py

```diff
@@ -141,25 +224,42 @@ def where(condition: _cpp.Variable, x: _cpp.Variable,
           y: _cpp.Variable) -> _cpp.Variable:
     """Return elements chosen from x or y depending on condition.
 
-    :param condition: Variable with dtype=bool. Where True, yield x, otherwise yield y.
-    :param x: Variable with values from which to choose.
-    :param y: Variable with values from which to choose.
-    :return: Variable with elements from x where condition is True, and elements from y
-             elsewhere.
-    :seealso: :py:func:`scipp.choose`
+    Parameters
+    ----------
+    condition:
+        Variable with dtype=bool.
+    x:
+        Variable with values from which to choose.
+    y:
+        Variable with values from which to choose.
+
+    Returns
+    -------
+    :
+        Variable with elements from x where condition is True
+        and elements from y elsewhere.
+
+    See Also
+    --------
+    scipp.choose
     """
     return _call_cpp_func(_cpp.where, condition, x, y)
 
 
-def to(var: Union[_cpp.Variable, _cpp.DataArray], *, unit=None, dtype=None, copy=True):
-    """
-    Converts a Variable or DataArray to a different dtype and/or a different unit.
+def to(
+        var: VariableLikeType,  # noqa
```

Comment:
is the `noqa` required because `var` is not on the line above?
Why not have `def to(var: VariableLikeType,` ?

Reply:
No, I put it because `var` is not documented and PyCharm doesn't like that. It makes no sense to document it because it becomes `self` after binding as a method.